listenbrainz/webserver/templates/user/import.html

```diff
@@ -58,4 +58,13 @@ <h3>Import a Last.fm backup file</h3>
       return false;
     }
   </script>
+  <script type="text/javascript">
+    function check() {
+      var i = document.getElementById("lastfm_input");
+      if(i.value == "")
```

Comment:
We don't have any explicit style checks here in the LB codebase, but please be careful about formatting of these if statements. Specifically:
```js
if (something) {
    x;
} else {
    y;
}
```
Always use braces, and be careful about whitespace